pysat/tests/test_instrument.py

```diff
@@ -1,24 +1,24 @@
 # -*- coding: utf-8 -*-
 # Test some of the basic _core functions
+import numpy as np
 import sys
-import pysat
-import pandas as pds
+
 from nose.tools import assert_raises, raises
 import nose.tools
+import pandas as pds
+
+import pysat
 import pysat.instruments.pysat_testing
-import numpy as np
+
 if sys.version_info[0] >= 3:
-    if sys.version_info[1] < 4:
-        import imp
-        reload = imp.reload
-    else:
-        import importlib
-        reload = importlib.reload
+    from importlib import reload as re_load
+else:
+    re_load = reload
 
 
 class TestBasics():
     def setup(self):
-        reload(pysat.instruments.pysat_testing)
+        re_load(pysat.instruments.pysat_testing)
```

Comment:
@rstoneback or @jklenzing , why is this necessary for a clean testing set up?

Reply:
I'm not sure.  @rstoneback?

Reply:
Adding to agenda for today's meeting.  This has been one of my questions for a while.

Reply:
I mess with the module in a couple tests, lines 210 and 222.

Reply:
Let's put some comments above this to that effect, so that people looking for code examples will know why this is done.

Reply:
That is reasonable. Issue created to remind us to more completely solve in the future.